cmake/std/atdm/sems-rhel7/environment.sh

```diff
@@ -282,4 +282,9 @@ export MPIF90=`which mpif90`
 
 export ATDM_CONFIG_MPI_PRE_FLAGS="--bind-to;none"
 
+# Install-related stuff
+export ATDM_CONFIG_TRIL_CMAKE_INSTALL_PREFIX_DATE_BASE_DEFAULT=/projects/atdm_devops/trilinos_installs
+export ATDM_CONFIG_MAKE_INSTALL_GROUP_DEFAULT=wg-run-as-atdm-devops
```

Comment:
Consider setting these once as default values in set_build_options.sh (line 59), instead of in every environment.sh file.

Reply:
The defaults `ATDM_CONFIG_TRIL_CMAKE_INSTALL_PREFIX_DATE_BASE_DEFAULT` and `ATDM_CONFIG_MAKE_INSTALL_GROUP_DEFAULT` are really for the ctest -S driver, not for regular users.